tracing-journald/src/lib.rs

```diff
@@ -155,6 +157,31 @@ impl Subscriber {
         &self.syslog_identifier
     }
 
+    /// Sets the syslog facility for this logger.
+    ///
+    /// The syslog facility comes from the classic syslog interface (`openlog()`
+    /// and `syslog()`). In syslog the facility argument is used to specify
+    /// what type of program is logging the message. This lets the configuration
+    /// file specify that messages from different facilities will be handled
+    /// differently. Systemd exposes it in the `SYSLOG_FACILITY` journal field,
+    /// and allows filtering log messages by syslog facility with `journalctl
+    /// --facility`.
+    ///
+    /// See [Journal Fields](https://www.freedesktop.org/software/systemd/man/systemd.journal-fields.html)
+    /// and [journalctl](https://www.freedesktop.org/software/systemd/man/journalctl.html)
+    /// for more information.
+    ///
+    /// If not set this field is left out.
+    pub fn with_syslog_facility(mut self, facility: SyslogFacility) -> Self {
+        self.syslog_facility = Some(facility);
+        self
+    }
+
+    /// Returns the syslog facility in use.
```

Comment:
nit, not important: perhaps worth adding
```suggestion
    /// Returns the syslog facility in use.
    ///
    /// A syslog facility can be set using [`Subscriber::with_syslog_facility`].
```

Reply:
Fixed, and also added to `syslog_identifier` for symmetry.